jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/events/TaskEventImpl.java

```diff
@@ -50,5 +53,10 @@ public TaskContext getTaskContext() {
 	public void setTaskContext(TaskContext context) {
 		this.taskContext = context;
 	}
+
+    @Override
+    public Date getEventDate() {
+        return this.eventDate;
+    }
```

Comment:
Maybe just nitpicking, but this looks like a bad formatting.

Reply:
reformatted